packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -24,6 +24,37 @@ MeetingsUtil.extractDestination = (destination, type) => {
   return dest;
 };
 
+/**
+ * Determine if the provided destination is a personId.
+ *
+ * @param {string} destination - The destination to validate.
+ * @returns {boolean} - True if the destination is a personId.
+ */
+MeetingsUtil.determineIfDestinationIsPerson = (destination) => {
+  const decodedId = Buffer.from(destination, 'base64')
+    .toString('ascii');
+
+  const destUUID = decodedId.split('/').pop();
+
+  return !!(
```

Comment:
Let's use the `deconstructHydraId` from @webex/common as well as the `hydraTypes` constant to handle this work.

Reply:
made this change.